benchmarks/synchrobench/src/main/java/com/oath/oak/synchrobench/contention/benchmark/ThreadLoopOak.java

```diff
@@ -90,7 +90,7 @@ public void stopThread() {
     public void run() {
 
         boolean change = Parameters.change;
-        int size = 100;
+        int size = 10000;
```

Comment:
Why to change?

Reply:
We wanted to run scans of 10K elements and not 100. 
I can change it back in the master it would be a 100.

Reply:
Please remain with 100

Reply:
Will do!